quickstart/docker/all-in-one/README.md

```diff
@@ -21,27 +21,27 @@ This is the primary use case for this project: running the `ziti edge quickstart
     docker compose pull
     ```
 
-2. Run it.
+1. Run it.
```

Comment:
@nicoalba is gonna have words with you over this some day!!! :) :D

Reply:
The linter was yelling at me :shrug: